### PR TITLE
updated & edited channels chapters

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10598
https://github.com/SUSE/spacewalk/issues/8952

### CODE REVIEW

File: modules/administration/pages/custom-channels.adoc [hunk 1]
```diff
@@ -35,23 +37,29 @@ Do not create child channels containing packages that are not compatible with th
 
 
 .Procedure: Creating a Custom Channel
-
 . In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and click btn:[Create Channel].
 . On the [guimenu]``Create Software Channel`` page, give your channel a name (for example, [systemitem]``My Tools SLES 15 SP1 x86_64``) and a label (for example, [systemitem]``my-tools-sles15sp1-x86_64``).
-Labels must not contain space characters or capital letters.
+Labels must not contain spaces or capital letters.
```
Comment (keichwa) [inline modules/administration/pages/custom-channels.adoc:42]: We usually call those uppercase letters. At least in German, "capitals" are a way to layout letters.

File: modules/administration/pages/custom-channels.adoc [hunk 2]
```diff
@@ -35,23 +37,29 @@ Do not create child channels containing packages that are not compatible with th
...
 . In the [guimenu]``Architecture`` drop down, choose the appropriate hardware architecture (for example, [systemitem]``x86_64``).
 . Provide any additional information in the contact details, channel access control, and GPG fields, as required for your environment.
 . Click btn:[Create Channel].
 
 
+[IMPORTANT]
+====
+By default, the ``Enable GPG Check`` field is checked when you create a new channel.
+If you would like to add custom packages and applications to your channel, make sure you uncheck this field to be able to install unsigned packages.
+Keep in mind this is a security risk for packages from an untrusted source.
```
Comment (keichwa) [inline modules/administration/pages/custom-channels.adoc:54]: Not sure whether we actually want sentence intros such as "Keep in mind".

File: modules/administration/pages/custom-channels.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/administration/pages/custom-channels.adoc:42]: Yeah, I was wondering about that.

File: modules/administration/pages/custom-channels.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/administration/pages/custom-channels.adoc:54]: Oof, no I missed that one.

**Comment by keichwa** [comment]
On 2020-05-27 11:15, Lana Brindley wrote:

> Oof, no I missed that one.

Good to see that we are nicely aligned ;-)

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
                                    Maxfeldstraße 5
                                    90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Pull request merged**